plugins/standard/applets/conference/conferences.js

```diff
@@ -0,0 +1,62 @@
+function join_call(conference_name){
+		
+	window.parent.Client.call({ 
+		'conference_name' : conference_name,
+		'muted' : 'false',
+		'beep' : 'true',
+		'Digits' : 1});
+	
+	return false;
+}
+
+function listen_call(conference_name){
+
+	window.parent.Client.call({ 
+		'conference_name' : conference_name,
+		'muted' : 'true',
+		'beep' : 'false',
+		'Digits' : 1});
+	
+	return false;
+}
+
+$(document).ready(function(){
+	jQuery(function($) {
+		$(function(){
+					
+			var
+			calls = {},
+			select = $('#calls'),
+			updateCalls = function() {
+				$.getJSON(window.location + '/?json=1', function(data) {
+					$.each(data, function(conference_name, call) {
+						if(!calls[conference_name]) {
+							calls[conference_name] = call;
+							
+							row = '<tr class="items-row" id="' + call.friendly_name + '">';
+							row += '<td>' + call.date_created + '</td>';
+							row += '<td>' + call.friendly_name + '</td>';
+							row += '<td>' + call.status + '</td>';
+							row += '<td><a onclick="join_call(\'' + call.friendly_name + '\');">Join</a></td>';
```

Comment:
The reason I didn't use $('#calls').on('click', '.listen', function(e) ... is because the version of jquery the project is using doesn't support the .on function.  As best I can tell the project is using jquery v1.6.2 and the .on function wasn't added to jquery until v1.7.  Version 1.7 was only released a few months after 1.6.2 so possibly upgrading jquery wouldn't be a breaking change.  Let me know how you want to proceed.